src/main/java/ch/njol/skript/variables/VariableStorage.java

```diff
@@ -33,7 +33,7 @@
  */
 // FIXME ! large databases (>25 MB) cause the server to be unresponsive instead of loading slowly
 @SuppressWarnings({"SuspiciousIndentAfterControlStatement", "removal"})
-public abstract class VariablesStorage implements Closeable {
+public abstract class VariableStorage implements Closeable {
```

Comment:
likewise i'm a bit averse to breaking change with no warning, but it's probably niche enough to be ok

Reply:
I thought this was already changed. Weird. Yes, it's fine, because the storages are a complete re-write.